src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/Loader/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const { slug = '' } = useParams();
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    setPeople(null);
+    getPeople()
+      .then((result) => setPeople(result))
+      .catch(() => setIsLoadingError(true))
+      .finally(() => setIsLoaded(true));
```

Comment:
It's not good to use an async function in a useEffect hook. So move it to a separate function and just call it in the useEffect